### PR TITLE
WV-2437: On mobile, category label should not show on "Featured" tab

### PR DESCRIPTION
**Pull request by victorsh**
## Description

When the "Featured" tab is chosen in a mobile layout, the header element which would display the category title in other tabs (e.g. Air Quality seen below under Hazards and Disasters tab) should not be visible.  Right now it just shows "All"

Fixes # WV-2437

## How To Test

- Click on "Add Layers"
- Click the header dropdown and selected "Featured"
- "All" should no longer be there.

Screen Shot 2023-03-08 at 2 12 35 PM

@nasa-gibs/worldview

### CODE REVIEW

File: web/js/modules/product-picker/reducers.js [hunk 1]
```diff
@@ -166,7 +166,8 @@ export function productPickerReducer(state = productPickerState, action) {
       return {
         ...state,
         categoryType: 'featured',
-        category,
+        // category,
```
Comment (Tomcariello) [inline web/js/modules/product-picker/reducers.js:169]: I would remove this comment at line 169.

**Review by Tomcariello — APPROVED** [review]
Works as described.

File: web/js/modules/product-picker/reducers.js (hunk 1 above)
Comment (victorsh) [reply web/js/modules/product-picker/reducers.js]: Removed

**Review by minniewong — APPROVED** [review]
Works now as expected!

**Pull request merged**